# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerDebugger.cpp
# Commit: 01fe98c08d10
# Full Hash: 01fe98c08d102c28e3a61209ea31834eff83202a
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-25 21:02:33
# Regressor Bug: 1899503
# File Overlap Count: 4
# Description:
#   Bug 1899503 - P3 PRemoteWorkerDebugger binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebugger binding.
#   
#   We are trying to keep the same logic debugger registration/unregistration to minimize the difference between the local debugger(the debugger on the content process main thread)
# ==============================================================================

diff -r 9bd2a82481f4 -r 01fe98c08d10 dom/workers/WorkerDebugger.cpp
--- a/dom/workers/WorkerDebugger.cpp	Tue Mar 25 09:13:20 2025 +0000
+++ b/dom/workers/WorkerDebugger.cpp	Tue Mar 25 09:13:20 2025 +0000
@@ -209,6 +209,14 @@
 }
 
 NS_IMETHODIMP
+WorkerDebugger::GetIsRemote(bool* aResult) {
+  AssertIsOnMainThread();
+
+  *aResult = false;
+  return NS_OK;
+}
+
+NS_IMETHODIMP
 WorkerDebugger::GetIsInitialized(bool* aResult) {
   AssertIsOnMainThread();
 